package.json

```diff
@@ -64,7 +64,7 @@
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@dotlottie/react-player": "^1.6.3",
-    "@expensify/react-native-live-markdown": "github:Expensify/react-native-live-markdown#f762be6fa832419dbbecb8a0cf64bf7dce18545b",
```

Comment:
@hoangzinh Yep, all necessary changes are already on Live Markdown's main.

Reply:
Unfortunately not, because 0.1.40 does not support new architecture

Reply:
but we can close this PR in favor of https://github.com/Expensify/App/pull/40102 which bumps Live Markdown to 0.1.47

Reply:
I think we just close this in favor of the PR https://github.com/Expensify/App/pull/40102 0.1.47 and then verify it's fixed there.

Reply:
I'm realizing though since we're making a lot of changes to react-native-live-markdown, we end up having multiple PRs that bump the version. We'll probably want a better way to handle as opposed to having each outdated minor version bump reviewed

Reply:
@thienlnam Fair point, we might want to batch updates on our side and just submit one PR with version bump that fixes multiple issues (unless there are other changes in E/App code). What do you think?

Reply:
I think that's a good idea - perhaps we group a few different bugs together so we don't end up with multiple PRs that are reviewing an outdated version